packages/frontend/src/components/EditorLayout/index.tsx

```diff
@@ -75,18 +79,19 @@ export default function EditorLayout(): React.ReactElement {
           justifyContent="space-between"
           alignItems="center"
           py={2}
-          px={2}
+          px={8}
           borderBottom="1px solid"
           borderColor="base.divider.subtle"
+          spacing={4}
         >
           <Box display="flex" flex={1} alignItems="center">
```

Comment:
nit: actually `Box display="flex"` is same as `Flex`. but doesn't matter if you change or not, this was old code. up to you.